daffodil-test/src/test/scala-new/edu/illinois/ncsa/daffodil/section13/packed/TestPacked.scala

```diff
@@ -54,4 +54,18 @@ class TestPacked {
   // @Test def testHexCharset03(): Unit = { runner.runOneTest("hexCharset03") } // textNumberPattern V symbol
   @Test def testHexCharset04(): Unit = { runner.runOneTest("hexCharset04") }
 
+  @Test def testPackedCharset01(): Unit = { runner.runOneTest("packedCharset01") }
+  @Test def testPackedCharset02(): Unit = { runner.runOneTest("packedCharset02") }
+  @Test def testPackedCharset03(): Unit = { runner.runOneTest("packedCharset03") }
```

Comment:
Does this test work? (packedCharset03) 

If so then binaryDecimalVirtualPoint works, because the test's schema has that, as "2", but there was review commentary elsewhere indicating that this feature isn't implemented?